.azuredevops/pipelineTemplates/jobs.validateModulePester.yml

```diff
@@ -149,6 +149,12 @@ jobs:
               Run        = @{
                 Container = New-PesterContainer -Path (Join-Path '$(moduleRepoRoot)' 'arm' '.global' 'global.module.tests.ps1') -Data @{
                   moduleFolderPaths = $moduleFolderPaths
+                  tokensToAvoid = @{
```

Comment:
tokensToAvoid sounds rude. Should we find a nicer name?

Reply:
Isn't that the idea? :D Open for suggestions though

Reply:
load it from a json file with "blacklist token names" :D

Reply:
You gave me the inspiration @MariusStorhaug 
```suggestion
                  notAllowedHardcodedTokens = @{
```

Reply:
`enforcedTokenList`?

Reply:
Fair enough

Reply:
Updated